hyphalnet/CPTAC_threshold_1.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 30 10:37:37 2020
+
+@author: linth
+"""
+import pandas as pd
+from scipy import stats
+data = pd.read_csv('CPTAC.csv')
+data.info()
+data.isnull().sum()
+data.head()
+# Creating protein dictionary
+aml_sample = data['AML sample']
+gene = data['Gene']
+log_fold = data['LogFoldChange']
+protein_dict = dict()
+n = len(aml_sample)
+for i in range(n):
+    protein_dict[aml_sample[i]]={gene[i]:log_fold[i]}

```

Comment:
This is the resulting dictionary we need. however, we need to ONLY have those proteins that are significant for each patient.